lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -242,6 +243,22 @@ def output_exclude_path(output_buffer, exclude_path, parent)
         regexp = exclude_path
         output_buffer.puts "    - !ruby/regexp /#{regexp.source}/"
       end
+
+      def safe?(config)
+        config['Safe'] || config['Safe'].nil?
+      end
+
+      def unsafe?(config)
+        config['Safe'] == false
+      end
+
+      def safe_autocorrect?(config)
+        config['SafeAutoCorrect'] || config['SafeAutoCorrect'].nil?
+      end
+
+      def unsafe_autocorrect?(config)
+        config['SafeAutoCorrect'] == false
+      end
```

Comment:
Looks like they can be combined into one method.
Actually, it's defined in autocorrect_logic, but namespace considerations will force us to redefine it here.
https://github.com/rubocop/rubocop/blob/00e7ac78fb83383ef537e9e08ecf2301a1427578/lib/rubocop/cop/autocorrect_logic.rb#L27-L29

```suggestion
      def safe_autocorrect?(config)
        config.fetch('Safe', true) && config.fetch('SafeAutoCorrect', true)
      end
```